mcs/tools/linker/Makefile

```diff
@@ -41,12 +49,48 @@ TEST_EXEC = MONO_PATH=$(LINKER_OUTPUT) $(RUNTIME) $(RUNTIME_FLAGS) --debug -O=-a
 
 .NOTPARALLEL:
 
-System/test-%.exe mscorlib/test-%.exe:
+System/test-%.exe mscorlib/test-%.exe link-all/link-%.exe link-sdk/link-%.exe:
 	@rm -rf $(LINKER_OUTPUT)
 	@mkdir $(LINKER_OUTPUT)
 	@echo Testing $@
 	$(LINKER) -a Tests/$@
 	$(TEST_EXEC) $(LINKER_OUTPUT)/$(@F)
 	@rm -rf $(LINKER_OUTPUT)
 
+BCL_ASSEMBLIES_CORE=mscorlib.dll System.dll System.Core.dll System.Xml.dll
```

Comment:
Why do we need the separation?

Reply:
Purely cosmetical so that the .csv and diff shows those assemblies first since they're the ones which have data and are the most interesting. Otherwise they're intermingled between a bunch of 0,0,0,0 values for assemblies which are completely linked away.